### PR TITLE
add support for BeginLoadQueryEvent and ExecuteLoadQueryEvent

### PR DESCRIPTION
**Pull request by jasonzzz**

### CODE REVIEW

**Comment by siddontang** [comment]
Hi @jasonzzz 

I am very interested that why you want to support these two events?

**Comment by jasonzzz** [comment]
If the user set `binlog_format=statement`, there's no available outputs when he `LOAD DATA INFILE ... INTO ...` in mysql.

File: replication/event.go [hunk 1]
```diff
@@ -299,6 +299,85 @@ func (e *GTIDEvent) Dump(w io.Writer) {
...
+	pos += 4
+
+	e.BlockData = data[pos:]
+
+    return nil
+}
+
+func (e *BeginLoadQueryEvent) Dump(w io.Writer) {
+	fmt.Fprintf(w, "File ID: %d\n", e.FileID)
+	fmt.Fprintf(w, "Block data: %s\n", e.BlockData)
```
Comment (siddontang) [inline replication/event.go]: The BlockData is human readable? Using hex print may be better?

Comment (jasonzzz) [inline replication/event.go]: Yes, the BlockData is human readable.

According to [BEGIN_LOAD_QUERY_EVENT](https://dev.mysql.com/doc/internals/en/begin-load-query-event.html), block-data is string.EOF, just like [QUERY_EVENT](https://dev.mysql.com/doc/internals/en/query-event.html) query.

**Comment by siddontang** [comment]
3Q

**Pull request merged**